src/storage/slotted_page.c

```diff
@@ -1489,6 +1490,32 @@ spage_find_empty_slot (THREAD_ENTRY * thread_p, PAGE_PTR page_p, int record_leng
   return SP_SUCCESS;
 }
 
+/*
+ * spage_move_slots() -
+ *   return:
+ *
+ *   start_slot_p(in): Pointer to slotted page pointer array
+ *   end_slot_p(in): Pointer to slotted page pointer array
+ *   offset(in): How much to move. Positive means go up in a slot page, and negative down.
+ *
+ *   Move [start_slot_p, end_slot_p) to the end_slot_p direction.
+ */
+static void
```

Comment:
Do we have a chance to move several offsets?
In aspects of code generalization, it is very good. But, how about just wrapping memmove() by spage_move_slots() with an inline keyword? Or how about just use memmove() right away?

Reply:
I thought It would be messy and unreadable due to the address arithmetic, but I've figured out it's not that complicated and even could be more readable. I accept your review and thank you.